tutorials/setup_tutorial.md

```diff
@@ -45,21 +45,21 @@ python -m nltk.downloader perluniprops nonbreaking_prefixes punkt
 
 ### Optional
 
-If you'll be using GPT, BERT, or other models supplied by `pytorch-pretrained-BERT`, then you may see speed gains from installing NVIDIA apex, following the instructions here: 
+If you'll be using GPT, BERT, or other models supplied by `pytorch-transformers`, then you may see speed gains from installing NVIDIA apex, following the instructions here:
```

Comment:
The changes to this PR will also have to change task tutorial as well.

Reply:
Done.